terraform/examples/daos_cluster/terraform.tfvars.perf.example

```diff
@@ -1,32 +1,40 @@
-project_id         = "<project_id>"
-region             = "us-central1"
-zone               = "us-central1-f"
-network            = "default"
-subnetwork_project = "<project_id>"
-subnetwork         = "default"
+project_id                 = "<project_id>"
```

Comment:
Is there anyway we can  say project_id = "MY_ID" on time then have subnetwork_project  server_os_project and any of its other uses default to the "project_id" var?   

If not maybe we should group all the project_id users together in this tfvars file it is really easy for the user to find them?  

Thoughts?

Reply:
I made changes to the server and client modules so that if the subnetwork projects are not specified (or have the default null vaule), then the project_id is used. This should be documented in the variable descriptions.

Reply:
After testing the new setups is very streamlined and I like using it :)